nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -18,13 +18,21 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <CoreCompileDependsOn>CsWinRTIncludeProjection;CsWinRTRemoveWinMDReferences;$(CoreCompileDependsOn)</CoreCompileDependsOn>
   </PropertyGroup>
   
-  <!--Remove Windows.Winmd reference to prevent compile collisions-->
-  <Target Name="CsWinRTRemoveWindowsReference" Outputs="@(Reference)" Condition="$(CsWinRTEnabled)">
+  <!-- Remove WinRT.Host.Shim.dll references -->
+  <Target Name="CsWinRTRemoveShimDllReferences" AfterTargets="ResolvePackageAssets" BeforeTargets="ResolveLockFileAnalyzers" Outputs="@(Reference)">
     <ItemGroup>
-      <Reference Remove="Windows"/>
+     <Reference             Remove="$(CsWinRTPath)lib\net5.0\WinRT.Host.Shim.dll"/>
+     <RuntimeCopyLocalItems Remove="$(CsWinRTPath)lib\net5.0\WinRT.Host.Shim.dll"/>
     </ItemGroup>
   </Target>
 
+  <!-- Remove Windows.Winmd reference to prevent compile collisions-->
+  <Target Name="CsWinRTRemoveWindowsReference" Outputs="@(Reference)" Condition="'$(CsWinRTEnabled)'=='true'">

```

Comment:
Is the == true change necessary. Does Condition="$(CsWinRTEnabled)" from before not work?

Reply:
Yep, reverted.